N/neper/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+using BinaryBuilder, Pkg
+
+name = "neper"
+version = v"4.5.0"
+
+sources = [
+    ArchiveSource("https://github.com/neperfepx/neper/archive/refs/tags/v$(version).tar.gz", "db80dd89e02207e9b056b05fb9fbe493199ce7c3736b2039104c595b4dcd02a9")
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/neper-*
+mkdir build && cd build
+CXXFLAGS="-lz"
+cmake -DCMAKE_INSTALL_PREFIX=$prefix \
+      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_CXX_FLAGS="${CXXFLAGS}" \
+      -DCMAKE_INSTALL_PREFIX_COMPLETION_FULL=$prefix/completion \
```

Comment:
I'd put this is in `share/` just not to have anything "non-standard" in the prefix, although this isn't much relevant for us
```suggestion
      -DCMAKE_INSTALL_PREFIX_COMPLETION_FULL=$prefix/share/completion \
```

Reply:
Makes sense. Turns out this isn't even needed for any of the working platforms since I got other errors down the line for other platforms.